src/linkify-matrix.ts

```diff
@@ -34,12 +36,18 @@ enum Type {
     GroupId = "groupid"
 }
 
-function parseFreeformMatrixLinks(linkify, token: '#' | '+', name: Type): void {
+/**
+ * Token should be one of the type of linkify.parser.TOKENS[AT | PLUS | POUND]
+ * but due to typing issues it's just not a feasible solution.
+ * This problem kind of gets solved in linkify 3.0
+ */
+function parseFreeformMatrixLinks(linkify, token: '#' | '+' | '@', type: Type): void {
```

Comment:
Can we abstract token behind an enum in that case?

Reply:
I guess it makes sense.